kinto-remote-settings/src/kinto_remote_settings/signer/listeners.py

```diff
@@ -308,7 +308,7 @@ def check_collection_status(
 
         # 2. work-in-progress -> to-review
         elif new_status == STATUS.TO_REVIEW:
-            if editors_group_uri not in user_principals:
+            if editors_group_uri not in user_principals and _to_review_enabled:
```

Comment:
Should we add a separate guard clause to with a message that reviews are disabled?

Reply:
I think we don't want that. This way when working on client scripts, they would work in all environments, including DEV where review is disabled